src/main/webapp/transcript.js

```diff
@@ -66,7 +66,7 @@ function appendTextToList(transcriptLine, ulElement) {
   const startTimestamp = `${startDate.getHours()}:${startDate.getMinutes()}:${
     startDate.getSeconds()}`;
   const endTimestamp =
-    `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
+      `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
```

Comment:
is this change a mistake?

Reply:
I needed to make this change for the js validation to work

Reply:
what do you mean by validation? the linter?

Reply:
Yes